Makefile

```diff
@@ -423,6 +385,17 @@ ifdef CODE_SIGN_CERTIFICATE_PASSWORD
 endif
 endif
 
+assets/osx/installer.tiff: assets/osx/installer.png assets/osx/installer@2x.png
+	tiffutil -cathidpicheck $^ -out $@
```

Comment:
That dependency comes with macOS (and its only used there). Do you think we should still document it?